pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # Minimum requirements for the build system to execute.
 # See https://github.com/scipy/scipy/pull/12940 for the AIX issue.
 requires = [
-    "setuptools>=38.6.0",
+    "setuptools>=51.0.0",
```

Comment:
do we also need to change environment.yaml (or was that a difffernt issue)?

Reply:
actually can we remove this now?

Reply:
The reason for the higher setuptools version was the ability to read certain information from setup.cfg, I think. So that's still something we need.

Reply:
good point ok then!